### DISCUSSION TITLE
[Question] How to simplify the italic() function in math environment?

### DISCUSSION THREAD

**Discussion by ysl2** [typst/typst#3850] [Q&A]
### Description

(This might be already answered, but in fact I didn't found any thread about this...)

There are so many `italic(.)` in my document, like this:

```
$
italic("Dice")(X,Y) = (2|X sect Y|) / (|X|+|Y|) \
italic("Jaccard")(X,Y) = (|X sect Y|) / (|X union Y|) \
italic("VS") = (2|X - Y|) / (|X| + |Y|)
$
```

Is there any method to simplify this?

### Use Case

1

**Comment by laurmaedje** [comment]
If you frequently write the same italic names, you could store them in variables:
```typ
#let dice = math.italic("Dice")
#let jaccard = math.italic("Jaccard")
#let vs = math.italic("VS")
$
  dice(X,Y) = (2|X sect Y|) / (|X|+|Y|) \
  jaccard(X,Y) = (|X sect Y|) / (|X union Y|) \
  vs = (2|X - Y|) / (|X| + |Y|)
$
```

---

PS: Best to use discussions rather than issues for questions (or ask on the [Discord](https://discord.gg/2uDybryKPe)). I have converted the issue into a discussion.
